rules/windows/command_and_control_remote_file_copy_mpcmdrun.toml

```diff
@@ -0,0 +1,45 @@
+[metadata]
+creation_date = "2020/09/03"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/09/03"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies MpCmdRun.exe Windows Defender configuration utility being used to download a remote file. Verify details such
+as parent process, url reputation and downloaded file details, in addition to that in case of download errors MpCmdRun
+logs this information automatically in the Appdata Temp folder in MpCmdRun.log where the download url is recorded.
+"""
```

Comment:
For the description, I like to identify what the rule detects, and potentially how an adversary could abuse what it's detecting.  Would you be open to putting some of the additional details in the `note` section?

> Verify details such as parent process, url reputation and downloaded file details, in addition to that in case of download errors MpCmdRun logs this information automatically in the Appdata Temp folder in MpCmdRun.log where the download url is recorded.

Reply:
moved to notes